versionedRouter.js

```diff
@@ -637,7 +637,9 @@ async function handleProxyRequest(destination, ctx) {
     }
   }
   ctx.body = { output: response };
-  ctx.status = response.status;
+  // Sending `204` status(obtained from destination) is not working as expected
+  // Since this is success scenario, we'll be forcefully sending `200` status-code to server
+  ctx.status = isHttpStatusSuccess(response.status) ? 200 : response.status;
```

Comment:
Was this change tested? I don't think it is defined or imported anywhere.

Reply:
It has been tested. The reason you don't see the import is because I stages only a part of the code. The other part I missed will add and commit it

Reply:
Please let me know if you find something missing. Thank you